### DISCUSSION TITLE
Theme_generator.py

### DISCUSSION THREAD

**Discussion by Alex-Wenner-FHR** [h2oai/wave#1036] [Q&A] [answered]
Hi! When I try to test the new theme_generator.py in the nightly pre-release, I see this error. I have installed the tar file and .whl file for the pre-release. 

![image](https://user-images.githubusercontent.com/78372056/135645516-1aa22338-8478-4459-9203-563c7cd83749.png)

**Comment by mturoci** [comment, marked as answer]
Hi @AlexWennerDT,

you need to pip install `.whl` file that is a part of every nightly release.

**Reply by Alex-Wenner-FHR** [reply]
I have done such! I can uninstall and try again to see if that helps!

**Reply by Alex-Wenner-FHR** [reply]
That resolved it, must have been a virtual env issue. Thanks!
